mayastor/src/bin/cli/rebuild_cli.rs

```diff
@@ -0,0 +1,272 @@
+                .help("uri of child to pause rebuilding"),
+        );
+
+    let resume = SubCommand::with_name("resume")
+        .about("resumes a rebuild")
+        .arg(
+            Arg::with_name("uuid")
+                .required(true)
+                .index(1)
+                .help("uuid of the nexus"),
+        )
+        .arg(
+            Arg::with_name("uri")
+                .required(true)
+                .index(2)
+                .help("uri of child to resume rebuilding"),
+        );
+
+    let state = SubCommand::with_name("state")
+        .about("gets the rebuild state of the child")
+        .arg(
+            Arg::with_name("uuid")
+                .required(true)
+                .index(1)
+                .help("uuid of the nexus"),
+        )
+        .arg(
+            Arg::with_name("uri")
+                .required(true)
+                .index(2)
+                .help("uri of child to get the rebuild state from"),
+        );
+
+    let progress = SubCommand::with_name("progress")
+        .about("shows the progress of a rebuild")
+        .arg(
+            Arg::with_name("uuid")
+                .required(true)
+                .index(1)
+                .help("uuid of the nexus"),
+        )
+        .arg(
+            Arg::with_name("uri")
+                .required(true)
+                .index(2)
+                .help("uri of child to get the rebuild progress from"),
+        );
+
+    SubCommand::with_name("rebuild")
+        .settings(&[
+            AppSettings::SubcommandRequiredElseHelp,
+            AppSettings::ColoredHelp,
+            AppSettings::ColorAlways,
+        ])
+        .about("Rebuild management")
+        .subcommand(start)
+        .subcommand(stop)
+        .subcommand(pause)
+        .subcommand(resume)
+        .subcommand(state)
+        .subcommand(progress)
+}
+
+async fn start(
+    mut ctx: Context,
+    matches: &ArgMatches<'_>,
+) -> Result<(), Status> {
+    let uuid = matches.value_of("uuid").unwrap().to_string();
+    let uri = matches.value_of("uri").unwrap().to_string();
+
+    ctx.v2(&format!(
+        "Starting rebuild of child {} on nexus {}",
+        uri, uuid
+    ));
+    ctx.client
+        .start_rebuild(rpc::StartRebuildRequest {
+            uuid: uuid.clone(),
+            uri: uri.clone(),
+        })
+        .await?;
+    ctx.v1(&format!(
+        "Started rebuild of child {} on nexus {}",
+        uri, uuid
+    ));
+    Ok(())
+}
+
+async fn stop(
+    mut ctx: Context,
+    matches: &ArgMatches<'_>,
+) -> Result<(), Status> {
+    let uuid = matches.value_of("uuid").unwrap().to_string();
+    let uri = matches.value_of("uri").unwrap().to_string();
+
+    ctx.v2(&format!(
+        "Stopping rebuild of child {} on nexus {}",
+        uri, uuid
+    ));
+    ctx.client
+        .stop_rebuild(rpc::StopRebuildRequest {
+            uuid: uuid.clone(),
+            uri: uri.clone(),
+        })
+        .await?;
+    ctx.v1(&format!(
```

Comment:
I wonder if these could be a bit misleading since it's not guaranteed that the action has been completed.

Reply:
Good point - I've changed the messages slightly to address this.